wordpress-mobile/MediaPicker-iOS
# Support non-image / video assets

This PR adds support for non-image and video assets. It goes alongside the https://github.com/wordpress-mobile/WordPress-iOS/tree/feature/media-library-filetypes branch for WPiOS.

For assets that aren't images or videos, it adds an icon and document file extension label to the media cells:

<img width="146" alt="screen shot 2017-04-26 at 15 41 21" src="https://cloud.githubusercontent.com/assets/4780/25440437/5c999aec-2a97-11e7-988d-75db637a9961.png">

and displays the filename in the 'caption' area at the bottom of the cell.

To test:

* The easiest way is probably to check out the [WPiOS branch](https://github.com/wordpress-mobile/WordPress-iOS/tree/feature/media-library-filetypes) that uses this PR's branch
* Ensure you have documents in a media library, and test that their icons display the new format
